packages/jetbrains-plugin/src/main/kotlin/com/mongodb/jbplugin/editor/CachedQueryService.kt

```diff
@@ -187,22 +187,22 @@ class CachedQueryService(
                 val readModel by query.source.project.service<DataGripBasedReadModelProvider>()
                 val buildInfo = readModel.slice(dataSource, BuildInfo.Slice)
 
-                queryWithDb.withTargetCluster(
```

Comment:
At some point we bugged this by not storing the modified query (data classes are immutable).